code/src/components/ListThoughts.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import moment from 'moment'
+import { LikeButton } from './LikeButton'
+import './listthoughts.css'
+
+// LISTING THE THOUGHTS FROM API (PROPS LET ME WORK WITH THE DATA WE'RE FETCHING IN THOUGHTS.JS)
+
+export const ListThoughts = (props) => {
+
+  return (
+
+    <div className="thoughts-wrapper">
+      {props.thoughts.map((thought) => (
+
+        <article className="thought" key={thought._id}>
+
+          <div className="message">{thought.message}</div>
+
+          <div className="details">
+            <div className="likes-wrapper">
+              {/* Passing the id and onThoughtLiked to Likebutton component */}
+              <LikeButton id={thought._id} onThoughtLiked={props.onThoughtLiked} hearts={thought.hearts} />

```

Comment:
Here you could add a `onClick()` that adds a class to make the heart to become pink!